### PR TITLE
Added performance statistics

### PR DESCRIPTION
**Pull request by hrueger**
Hi,
I added a performance statistics feature. It updates every second and looks like this:
grafik
I also added some semicolons for a consistent code style.

In case you want to this repo to just show the basic usage without any additional features, I can also create a new repo and put all my experiments there (I'm working on projectors / preview / etc. )

### CODE REVIEW

File: obsRecorder.js [hunk 1]
```diff
@@ -1,22 +1,27 @@
 const path = require('path');
 const { Subject } = require('rxjs');
 const { first } = require('rxjs/operators');
+const { ipcMain } = require('electron');
```
Comment (Envek) [inline obsRecorder.js]: I can't find any usages of it. Is it needed?

File: obsRecorder.js [hunk 2]
```diff
@@ -1,22 +1,27 @@
...
 
 const osn = require("obs-studio-node");
 
 let obsInitialized = false;
 
 // Init the library, launch OBS Studio instance, configure it, set up sources and scene
-function initialize() {
+function initialize(webContents) {
```
Comment (Envek) [inline obsRecorder.js:10]: It feels somewhat wrong for me, but I can't propose any better way to do it for now.

**Review by Envek — COMMENTED** [review]
Any improvements are welcome!

When it start to feel too complicated we will think on how to simplify it or write comprehensive manual.

File: obsRecorder.js (hunk 1 above)
Comment (hrueger) [reply obsRecorder.js]: No, you're right. I forgot to delete that line.

File: obsRecorder.js (hunk 2 above)
Comment (hrueger) [reply obsRecorder.js:10]: I know, I tried to require it somehow, but the `remote` module is `undefined` when required in this context...

**Comment by Envek** [comment]
Great! Thank you very much!

**Pull request merged**